src/store/index.js

```diff
@@ -1,4 +1,44 @@
 import { createStore } from 'vuex'
+import app from '@/helpers/axiosConfig'
+
+const pokemons = {
+  namespaced: true,
+  state: {
+    list: [],
+    pokemon: null,
+    abilities: [],
+  },
+  mutations: {
+    addPokemon(state, pokemon) {
+      const pokemons = JSON.parse(localStorage.getItem('pokemons'))
```

Comment:
If you plan to move these out from the localStorage to let's say webSQL, indexedDB, etc, then it will become async and couldn't fit into the mutations section and should be moved to an action instead.

Reply:
Also why do you use localstorage here? The swrv lib should be responsible for caching all these and the others can just be plain properties.

Reply:
just because I did not know about swrv at this time.